OWML.Common/ConsoleUtils.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace OWML.Common
+{
+    public static class ConsoleUtils
```

Comment:
whats the reason for this class?
can't its method be part of ModConsole or one of its children?
if it must exist - must it be static?

Reply:
this code was duplicated in program, socketlistener and outputwriter

Reply:
isn't it easier to have a static class you can just use than making a `new` every time you want to use it?

Reply:
it's great that you collected duplicated code to one place 👍
looks like all places this is used is from Launcher? in that case, consider putting this file there.
also, looks like this code can be put inside OutputWriter.cs? i know it's not in Launcher, but i guess it's only used from there - if so you can consider doing both: put this code inside OutputWriter and move it to Launcher. 
hard to properly check all this from phone 😋

Reply:
hypothesis: this belongs in OutputWriter.WriteLine. it must be newed up early and used (re-used - the same object) all the places you identified. i'm unable to verify that this makes 100% sense

Reply:
it's only using in program and socketlistener for writing errors/connection status to the console... could be removed?